app/controllers/reviews_controller.rb

```diff
@@ -30,6 +30,13 @@ def update
     redirect_to "/items/#{review.item.id}"
   end
 
+  def destroy
+    #binding.pry
```

Comment:
can we remove this if it's no longer needed before pushing into master - just so keep master as clean as possible